### PR TITLE
[EVNT-238] Add Authorization header

### PR DESCRIPTION
**Pull request by jpramos123**

### CODE REVIEW

**Review by fellipeh — APPROVED** [review]

**Review by CodeHeeler — APPROVED** [review]

File: splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java [hunk 1]
```diff
@@ -140,14 +143,14 @@ private void configureHandler() throws Exception {
                         .authenticationPreemptive(true)
                         .authMethod("Basic")
                         .httpMethod("POST")
-                        .authUsername("x")
+                        .authUsername("unused")
```
Comment (beav) [inline splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java:146]: is this still needed if you are setting authorization above?

Comment (jpramos123) [reply splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java:146]: Good point. No there is not need to keep this!

**Comment by beav** [comment]
I dont have an environment to test this, but it seems OK to me from eyeball review

**Review by beav — APPROVED** [review]

File: splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java [hunk 2]
```diff
@@ -121,6 +121,9 @@ private void configureHandler() throws Exception {
             //Add headers useful for error reporting and metrics
             .setHeader("targetUrl", simple("${headers.metadata[url]}"))
             .setHeader("timeIn", simpleF("%d", System.currentTimeMillis()))
+            
+            //Set Authorization header
+            .setHeader("Authorization", simpleF("Splunk %s", "${headers.metadata[X-Insight-Token]}"))
```
Comment (vkrizan) [inline splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java:126]: @jpramos123 are you sure that this sets the HTTP header? The setHeader sets header on Camel's message exchange.
Here are headers that the Camel HTTP component reads: https://camel.apache.org/components/3.15.x/http-component.html#_message_headers

Have a look at #27, whether that fixes the issue.

**Pull request merged**